README.md

```diff
@@ -1,12 +1,12 @@
 # Zulip overview
 
-Zulip is a powerful, open source group chat application that combines the
-immediacy of real-time chat with the productivity benefits of threaded
-conversations. Zulip is used by open source projects, Fortune 500 companies,
-large standards bodies, and others who need a real-time chat system that
-allows users to easily process hundreds or thousands of messages a day. With
-over 500 contributors merging over 500 commits a month, Zulip is also the
-largest and fastest growing open source group chat project.
+Zulip is a powerful, free and open source group chat application that combines
```

Comment:
The word "free" is confusing here, when not paired with "software". The natural meaning of it is "gratis", which isn't the right meaning.

Reply:
But "free and open source software group chat" doesn't sound right either.

Reply:
Indeed. Some more creative edit will be needed.

Reply:
@gnprice what about
> Zulip is a powerful, free/libre and open source group chat application that combines ...

This way, the meaning is anchored by the addition of 'libre', and there is no need to add 'software'.